docs/well-known-project-properties.md

```diff
@@ -173,6 +173,49 @@ Specifies whether the contents of the _Application Designer_ folder are only vis
 <PropertyGroup>
 ```
 
+#### __DesignerFunctionVisibility (enum)__
```

Comment:
Consider linking to the equivalent VS SDK enums from docs.microsoft.com, and mentioning that these values map directly to them.